src/js/components/DataTableColumns/stories/Inline.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { Data, DataFilters, DataTable, Grid } from 'grommet';
+
+import { DataTableColumns } from '../DataTableColumns';
+import { columns, DATA } from '../../DataTable/stories/data';
+
+export const Inline = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Grid pad="large" columns={['large']} justifyContent="center">
+    <Data data={DATA} updateOn="change">
+      <DataFilters>
+        <DataTableColumns
```

Comment:
<img width="811" alt="Screen Shot 2023-01-03 at 12 39 25 PM" src="https://user-images.githubusercontent.com/54560994/210429095-26645acb-4961-424e-ba72-4e0889b99e9d.png">

In the grommet theme it looks like there isn't enough spacing between the search and the CheckBoxGroup or the CheckBoxGroup and the DataTable